prometheus/prometheus.go

```diff
@@ -82,6 +84,44 @@ func (p *Prometheus) QueryRange(query string, start, end time.Time, step time.Du
 	return v, nil
 }
 
+// QueryRangeAggregation returns the aggregation from the given query
+// if the query returns multiple timeseries, their data points are aggregated into a single one
+func (p *Prometheus) QueryRangeAggregatedTS(query string, start, end time.Time, step time.Duration, aggregation Aggregation) (float64, error) {
+	var err error
+	var datapoints []float64
+	var result float64
+	r := apiv1.Range{Start: start, End: end, Step: step}
+	v, _, err := p.api.QueryRange(context.TODO(), query, r)
```

Comment:
can we leverage the `QueryRange` method already present in this same file for querying?